src/css-style-value.js

```diff
@@ -70,5 +70,15 @@
     return styleValueArray;
   };
 
+
+  internal.CSSStyleValue = function(cssText) {
+    if (!cssText) {
+      throw new TypeError('CSSStyleValue must have a value');
+    }
+    this.cssText = cssText;
+  }
+  internal.inherit(internal.CSSStyleValue, CSSStyleValue);
```

Comment:
I think we should change this back to just setting the prototype, so that the constructor is fully hidden.